.travis.yml

```diff
@@ -7,6 +7,9 @@ stages:
   - check
   - build
 
+python:
+  - "3.7"
```

Comment:
This doesn't do what you intended, as you can see in the build logs Python 2.7 is still being used. Instead you will need to do one of the following:

* during `install` phase add command `pyenv global system 3.7` (probably not best option IMO)
* add a `.python-version` file with `3.7` as contents to the root of this repo
* add to `env` the `PYENV_VERSION` variable with `3.7` value

You may also need to use `pip3` instead of `pip` to install scons. Also, unless you use the first option, `sudo` should be unnecessary.

Reply:
The correction can be simpler thanks for pointing this out, swapping out pip for pip3 will mean it will use a pip3 variant of scons, which has python3 shebang.

Reply:
After re-reading documentation you're right we would need the python specifier and lang set to python.

I think there might be an easier way will do a bit of research